.github/go-version

```diff
@@ -0,0 +1 @@
+1.23
```

Comment:
Need to newline at the end of file. (you can see red circle mark)
https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

Reply:
oops, fixed.